setup.py

```diff
@@ -71,7 +71,7 @@ def get_static_file_paths():
 
 setup(
     name='toucan_connectors',
-    version='1.3.29',
+    version='1.3.30',
```

Comment:
```suggestion
    version='1.3.31',
```
Because of this [PR](https://github.com/ToucanToco/toucan-connectors/pull/524)

Reply:
yup